README.md

```diff
@@ -0,0 +1,320 @@
+# `bpatch`
+
+**PURPOSE:** Suppose you need to modify a file and replace several byte sequences with other byte sequences. Furthermore, imagine you want these modification rules to be formalized in a human-readable format, so they're easy to edit. If that's the case, then this tool is exactly what you need.
+
+Domains where `bpatch` will be surely useful
```

Comment:
I recommend to decouple domains of bpatch as utility and domains of bpatch as static library. 
Also choosing bpatch as static library I want to see some proofs or let's say advantages why it better than others stream editors like seg.